src/Compilers/CSharp/Portable/Symbols/Source/SourceMethodSymbol.cs

```diff
@@ -1566,10 +1566,13 @@ protected void CheckModifiersForBody(SyntaxNode declarationSyntax, Location loca
 
                 if (!ContainingAssembly.RuntimeSupportsDefaultInterfaceImplementation)
                 {
+                    // PROTOTYPE(DefaultInterfaceImplementation): It looks like some flavor of static members was supported by
+                    //                                            legacy runtimes. Should we suppress this error for them?
                     diagnostics.Add(ErrorCode.ERR_RuntimeDoesNotSupportDefaultInterfaceImplementation, location);
```

Comment:
>diagnostics [](start = 20, length = 11)

I believe static methods in interfaces are permitted in current VMs as well. If that is true, we should not complain about them.